src/encoded/tests/test_search.py

```diff
@@ -137,6 +137,32 @@ def test_search_with_simple_query(workbook, testapp):
     assert not set(mouse_uuids).issubset(set(mauxz_uuids))
 
 
+def test_search_ngram(workbook, testapp):
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [redefinition of unused 'workbook' from line 2](https://app.codacy.com/manual/4dn/fourfront/pullRequest?prid=4756039)